origin-dapp/src/components/onboarding.js

```diff
@@ -95,6 +96,7 @@ class Onboarding extends Component {
 
     // To Do: handle incoming messages when no Origin Messaging Private Key is available
     origin.messaging.events.on('emsg', obj => {
+      analytics.event('Notifications', 'ErrorNoDecryption')
```

Comment:
@DanielVF this is going to produce a lot of noise. 🙉 I'm not sure that we care about this, but I guess we can filter it out?

Reply:
Yeah, you can comment it out. I was just putting analytics on things I saw happen.